model/fieldtypes/compat/autoload.php

```diff
@@ -8,6 +8,7 @@ function php5_compat_autoloader($classname) {
 	$classMap = array(
 		"int" => "/framework/model/fieldtypes/compat/Int.php",
 		"float" => "/framework/model/fieldtypes/compat/Float.php",
+        "object" => "/framework/core/compat/Object.php",
```

Comment:
I think we need another `if (PHP_MINOR_VERSION...) {` to check specifically for PHP 7.2 for this class. Otherwise this will unnecessarily break things for people using 7.0 and 7.1

Reply:
good point

Reply:
good point